Tutorials/Editors-Manual/Getting-Started-With-Umbraco/Creating-Saving-and-Publishing-Content/index.md

```diff
@@ -98,6 +98,48 @@ To set up scheduled unpublishing, follow these steps:
     ![Schedualed unpublishing.](images/Schedule_Unpublishing_v9.png)
 5. Select **Schedule**.
 
+## Comparing Content between environments
+
+:::note
+Content comparison is currently available in all Umbraco Cloud projects running Umbraco Deploy 9.3+ for Umbraco 9 and Umbraco Deploy 4.5+ for Umbraco 8.
```

Comment:
Try to avoid using words like "currently", as it makes it sounds like this will be changed at some point.

Also, is is necessary to mention to specific version numbers of Deploy?
Remember that this document is intended for Content Editors, and they will usually have nothing to do with versionng.